src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -102,6 +102,10 @@ public function preSubmit(FormEvent $event)
         $form = $event->getForm();
         $data = $event->getData();
 
+        if ($data instanceof \Traversable) {
```

Comment:
Hi @ybensacq seems you missed to fix one of @dosten comment here.

Should be `if ($data instanceof \Traversable && $data instanceof \ArrayAccess)`